app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -254,7 +254,7 @@ protected function updateTimelineAction(Request $request)
         $includeEvents = InputHelper::clean($request->request->get('includeEvents', []));
         $excludeEvents = InputHelper::clean($request->request->get('excludeEvents', []));
         $search        = InputHelper::clean($request->request->get('search'));
-        $leadId        = InputHelper::int($request->request->get('leadId'));
+        $leadId        = (int) $request->request->get('leadId');
```

Comment:
This isn't a bug per se. It works because of the magic method __callStatic here https://github.com/mautic/mautic/blob/97d13d23d728ac794beac1a37837eca4d863bbad/app/bundles/CoreBundle/Helper/InputHelper.php#L108